docs/src/reference/python/data.rst

```diff
@@ -0,0 +1,5 @@
+Data arrays
+===========
+
+.. autoclass:: equistore.data.Array
+    :members:
```

Comment:
for some reason this renders as empty? I though we had docs for this, even if they are minimal

Reply:
Weird. It renders locally fine. I changed it to exactly the same line as before the PR. Lets see what RTD is doing now...

Reply:
Hmm still seems to render properly. A reason might be that the docs are only injected if `HAS_TORCH=True` in `python/src/equistore/data/extract.py`. But why did it worked before...